concepts/60 Themes and Styles/05 Predefined Themes/50 Apply a Theme.md

```diff
@@ -1,9 +1,9 @@
 Do the following to apply a theme:
 
-- **jQuery, Knockout, AngularJS**   
-Include `dx.common.css` and a theme stylesheet in the `<head>` tag on your index page. You can use [local files](/concepts/Common/Distribution%20Channels/15%20ZIP%20Archive.md '/Documentation/Guide/Common/Distribution_Channels/ZIP_Archive/') or get the stylesheets [from CDN](/concepts/Common/Distribution%20Channels/05%20CDN '/Documentation/Guide/Common/Distribution_Channels/CDN/').
+- **jQuery**   
+Include `dx.common.css` and a theme stylesheet in the `<head>` tag on your index page. You can use [local files](/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/#Local_Files) or get the stylesheets [from CDN](/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/#CDN_Files).
 
-- **Angular, React, or Vue**    
+- **Angular, React, Vue**    
 Configure the module bundler you are using and import `dx.common.css` and a theme stylesheet. See the following instructions: [Angular](/concepts/40%20Angular%20Components/10%20Getting%20Started/03%20Add%20DevExtreme%20to%20an%20Angular%20CLI%20Application/03%20Configure%20Stylesheets.md '/Documentation/Guide/Angular_Components/Getting_Started/Add_DevExtreme_to_an_Angular_CLI_Application/#Configure_Stylesheets'), [React](/concepts/50%20React%20Components/05%20Add%20DevExtreme%20to%20a%20React%20Application/40%20Import%20Stylesheets.md '/Documentation/Guide/React_Components/Add_DevExtreme_to_a_React_Application/#Import_Stylesheets'), [Vue](/concepts/55%20Vue%20Components/05%20Add%20DevExtreme%20to%20a%20Vue%20Application/40%20Import%20Stylesheets.md '/Documentation/Guide/Vue_Components/Add_DevExtreme_to_a_Vue_Application/#Import_Stylesheets').
```

Comment:
Let's avoid the "using" gerund. Can we say:
Configure the current module bundler, and import `dx.common.css` and a theme stylesheet.

Reply:
```suggestion
Configure the module bundler you use and import `dx.common.css` and a theme stylesheet. See the following instructions: [Angular](/concepts/40%20Angular%20Components/10%20Getting%20Started/03%20Add%20DevExtreme%20to%20an%20Angular%20CLI%20Application/03%20Configure%20Stylesheets.md '/Documentation/Guide/Angular_Components/Getting_Started/Add_DevExtreme_to_an_Angular_CLI_Application/#Configure_Stylesheets'), [React](/concepts/50%20React%20Components/05%20Add%20DevExtreme%20to%20a%20React%20Application/40%20Import%20Stylesheets.md '/Documentation/Guide/React_Components/Add_DevExtreme_to_a_React_Application/#Import_Stylesheets'), [Vue](/concepts/55%20Vue%20Components/05%20Add%20DevExtreme%20to%20a%20Vue%20Application/40%20Import%20Stylesheets.md '/Documentation/Guide/Vue_Components/Add_DevExtreme_to_a_Vue_Application/#Import_Stylesheets').
```

Reply:
For the first sentence, let's add a comma:
Configure the module bundler you use, and import `dx.common.css` and a theme stylesheet.